config: update config descriptions to match docs

Deflaimun opened the pull request:
Backfeed changes to configuration descriptions that were added to docs. 

## Backports Required

<!-- Checking at least one of the checkboxes is REQUIRED if this PR is not a backport. -->

- [x] none - not a bug fix
- [ ] none - this is a backport
- [ ] none - issue does not exist in previous branches
- [ ] none - papercut/not impactful enough to backport
- [ ] v24.2.x
- [ ] v24.1.x
- [ ] v23.3.x

## Release Notes

### Improvements

* Improve property configuration descriptions. 

<!--
If the changes in this PR do not need to be mentioned in the release
notes, then don't add a sub-section and simply list `none`, e.g.

* none

Otherwise, adding a sub-section or `none` is REQUIRED if the PR is not a backport PR.
If this is a backport PR, adding contents to this section will override
the release notes section inherited from the original PR to dev.

Add one or more of the sub-sections with a short description bullet
point of the change, e.g.

### Bug Fixes

* Short description of the bug fix if this is a PR to `dev` branch.

### Features

* Short description of the feature. Explain how to configure.

### Improvements

* Short description of how this PR improves existing behavior.

-->

dotnwat commented on src/v/config/node_config.cc:
```diff
@@ -149,73 +155,69 @@ node_config::node_config() noexcept
+
+* Disabled - Redpanda does not start in FIPS mode.
+
+* Permissive - Redpanda performs the same check as enabled, but a warning is logged, and Redpanda continues to run. Redpanda loads the OpenSSL FIPS provider into the OpenSSL library. After this completes, Redpanda is operating in FIPS mode, which means that the TLS cipher suites available to users are limited to the TLSv1.2 and TLSv1.3 NIST-approved cryptographic methods.
```
* we try to line wrap around 80 columns
* maybe we need to maintain the human formatted version somewhere else?

Deflaimun replied:
It would be awesome if we could move those messages to a message bundle of sorts.

Feediver1 commented:
@dotnwat @Deflaimun Can we get a status update please?

aanthony-rp commented on src/v/config/node_config.cc:
```diff
@@ -26,108 +26,114 @@ node_config::node_config() noexcept
-      "List of the seed servers used to join current cluster. If the "
-      "seed_server list is empty the node will be a cluster root and it will "
-      "form a new cluster",
+      "List of the seed servers used to join current cluster. If the `seed_servers` list is empty the node will be a cluster root and it will form a new cluster.
```
Unfortunately, multi-line rich formatting like this will cause UX issues because we do output these strings in certain scenarios where we expect the string to be a one-line string.  

@dotnwat and I had an idea that we could extend the C++ to have a   _desc   and a _longdesc   member so that we could safely integrate really nice summaries like this into the product.   

summary for today:  any changes coming into this file need to be one-line strings.  We'll need to follow up to accept larger blobs like this one.  I'm happy to get on a call to help clarify what needs to change.

aanthony-rp commented on src/v/config/node_config.cc:
```diff
@@ -26,108 +26,114 @@ node_config::node_config() noexcept
-                "Duplicate items in seed_servers: {}", *s_dupe_i);
-          }
-          return std::nullopt;
+    std::sort(s.begin(), s.end());
```
I don't think you meant to change this, looks like a formatter did it.

Deflaimun replied:
Yes, that's correct.

aanthony-rp commented on src/v/config/node_config.cc:
```diff
@@ -26,108 +26,114 @@ node_config::node_config() noexcept
-      "seed_servers list must be identical among those nodes' configurations, "
-      "and those nodes will form the initial cluster.",
+      "Controls how a new cluster is formed. All brokers in a cluster must have the same value.
+
```
even in cases like this, we don't want to have line breaks.

aanthony-rp commented on src/v/config/node_config.cc:
```diff
@@ -26,108 +26,114 @@ node_config::node_config() noexcept
       *this,
       "admin_api_tls",
-      "TLS configuration for admin HTTP server",
+      "Specifies the TLS configuration for the HTTP Admin API.",
```
changes like this are 100% compatible.

aanthony-rp reviewed (changes requested):
We do have some functionality problems caused by several changes within.  Please see my comments for more details. briefly:  any change that remains as a single line of text with no breaks will be accepted.  Any multi-line doc strings cannot be accepted without a code change.   let's work together on next steps.

Deflaimun commented:
- Force-push: Updated peer code review. Removed multi-line strings. For some specific cases, added the guide to check docs "For more information see [link]"

dotnwat commented:
@Deflaimun i've fixed the compilation error and formatting. lemme know if it looks ok.

dotnwat commented on src/v/config/configuration.cc:
```diff
@@ -1654,35 +1799,44 @@ configuration::configuration()
-      "Optional API endpoint",
+      "Optional API endpoint. - AWS: When blank, this is automatically "
+      "generated using <<cloud_storage_region,region>> and "
+      "<<cloud_storage_bucket,bucket>>. Otherwise, this uses the value "
```
> <<cloud_storage_bucket,bucket>>

is this markdown formatting?

dotnwat commented on src/v/config/node_config.cc:
```diff
@@ -44,15 +46,35 @@ node_config::node_config() noexcept
-      "form a new cluster",
+      "`seed_servers` list is empty the node will be a cluster root and it "
+      "will form a new cluster. When `empty_seed_starts_cluster` is `true`, "
+      "Redpanda enables one broker with an empty `seed_servers` list to "
```
> `seed_servers`

this isn't a big deal, but these strings really aren't intended to contain markdown, and i'm assuming that is what the backticks are for.

vbotbuildovich commented:
<!--NEW FAILURES-->
new failures in https://buildkite.com/redpanda/redpanda/builds/55693#01924fe1-7644-44b7-a138-7d978b531e44:
```
"rptest.tests.cluster_config_test.ClusterConfigTest.test_dry_run"
"rptest.tests.connection_limits_test.ConnectionLimitsTest.test_null"
"rptest.tests.cluster_config_test.ClusterConfigTest.test_validate_cloud_storage_cache_size_config"
"rptest.tests.cluster_features_test.FeaturesUpgradeActivationTest.test_new_cluster_only_activation.upgrade=True"
"rptest.tests.rbac_test.RBACTelemetryTest.test_telemetry"
"rptest.tests.cpu_stress_injection_test.CpuStressInjectionTest.test_stress_fibers_spins"
"rptest.tests.storage_resources_test.StorageResourceRestartTest.test_recovery_reads.acks=-1.clean_shutdown=False"
"rptest.tests.retention_policy_test.BogusTimestampTest.test_bogus_timestamps.mixed_timestamps=False.use_broker_timestamps=False"
"rptest.tests.storage_resources_test.StorageResourceTest.test_files_and_fds"
"rptest.tests.mixed_self_test.RedpandaMixedTestSelfTest.test_metrics"
"rptest.tests.log_level_test.LogLevelTest.test_log_level_multiple_expiry.loggers=.admin_api_server.raft"
"rptest.tests.quota_management_test.QuotaManagementTest.test_describe_multiple_components.strict=False"
"rptest.tests.index_recovery_test.IndexRecoveryTest.index_recovers_test"
"rptest.tests.kafka_cli_client_compat_test.KafkaCliClientCompatTest.test_create_bad_acl"
"rptest.tests.crash_loop_checks_test.CrashLoopChecksTest.test_crash_loop_with_misconfiguration"
"rptest.tests.read_replica_e2e_test.TestMisconfiguredReadReplicaService.test_create_replica_without_cloud_enabled"
"rptest.tests.quota_management_test.QuotaManagementTest.test_kafka_configs"
"rptest.tests.retention_policy_test.ShadowIndexingCloudRetentionTest.test_cloud_size_based_retention_application.cloud_storage_type=CloudStorageType.S3"
"rptest.tests.retention_policy_test.ShadowIndexingLocalRetentionTest.test_shadow_indexing_default_local_retention.cluster_remote_write=False.topic_remote_write=-1.cloud_storage_type=CloudStorageType.ABS"
"rptest.tests.retention_policy_test.ShadowIndexingLocalRetentionTest.test_shadow_indexing_default_local_retention.cluster_remote_write=False.topic_remote_write=true.cloud_storage_type=CloudStorageType.ABS"
"rptest.tests.retention_policy_test.ShadowIndexingLocalRetentionTest.test_local_time_based_retention_is_overridden.local_retention_ms=-1.cloud_storage_type=CloudStorageType.ABS"
"rptest.tests.rpk_acl_test.RpkACLTest.test_create_update"
"rptest.tests.retention_policy_test.ShadowIndexingLocalRetentionTest.test_shadow_indexing_default_local_retention.cluster_remote_write=True.topic_remote_write=false.cloud_storage_type=CloudStorageType.ABS"
"rptest.tests.rpk_acl_test.RpkACLTest.test_role_acl"
"rptest.tests.retention_policy_test.ShadowIndexingLocalRetentionTest.test_shadow_indexing_non_default_local_retention.cloud_storage_type=CloudStorageType.ABS"
"rptest.tests.schema_registry_test.SchemaRegistryCompatibilityModes.test_compatible_schemas.schema_type=AVRO.mode=BACKWARD_TRANSITIVE"
"rptest.tests.rpk_topic_test.RpkToolTest.test_create_topic_with_invalid_config.config_type=cleanup.policy"
"rptest.tests.schema_registry_test.SchemaRegistryCompatibilityModes.test_compatible_schemas.schema_type=AVRO.mode=FULL_TRANSITIVE"
"rptest.tests.schema_registry_test.SchemaRegistryCompatibilityModes.test_compatible_schemas.schema_type=JSON.mode=FORWARD"
"rptest.tests.schema_registry_test.SchemaRegistryCompatibilityModes.test_compatible_schemas.schema_type=JSON.mode=NONE"
"rptest.tests.schema_registry_test.SchemaValidationTopicPropertiesTest.test_schema_id_validation_active_config.mode=SchemaIdValidationMode.REDPANDA"
"rptest.tests.schema_registry_test.SchemaValidationTopicPropertiesTest.test_schema_id_validation_active_nondefault_config.mode=SchemaIdValidationMode.REDPANDA"
"rptest.tests.schema_registry_test.SchemaValidationTopicPropertiesTest.test_schema_id_validation_disabled_config.mode=SchemaIdValidationMode.REDPANDA"
"rptest.tests.rpk_group_test.RpkGroupCommandsTest.test_group_seek_to_group"
"rptest.transactions.idempotency_test.IdempotentProducerRecoveryTest.test_librdkafka_recovery_on_producer_eviction"
"rptest.tests.acls_test.AccessControlListAuthzTest.test_alter_quotas"
"rptest.tests.topic_creation_test.CreateSITopicsTest.test_shadow_indexing_mode"
"rptest.tests.archive_retention_test.CloudArchiveRetentionTest.test_delete.cloud_storage_type=CloudStorageType.ABS.retention_type=retention.ms"
"rptest.tests.audit_log_test.AuditLogTestKafkaApi.test_management"
"rptest.tests.audit_log_test.AuditLogTestKafkaAuthnApi.test_authn_messages"
"rptest.tests.read_replica_e2e_test.TestReadReplicaService.test_simple_end_to_end.partition_count=10.cloud_storage_type_and_url_style=.CloudStorageType.S3.1.path"
"rptest.tests.mirror_maker_test.TestMirrorMakerService.test_simple_end_to_end.source_type=kafka"
"rptest.tests.audit_log_test.AuditLogTestSchemaRegistry.test_sr_audit_bad_authn"
"rptest.tests.audit_log_test.AuditLogTestsAppLifecycle.test_recovery_mode"
"rptest.tests.partition_move_interruption_test.PartitionMoveInterruption.test_cancelling_partition_move.replication_factor=3.unclean_abort=False.recovery=no_recovery.compacted=False"
"rptest.tests.compatibility.kafka_streams_test.KafkaStreamsSessionWindow.test_kafka_streams"
"rptest.tests.partition_move_interruption_test.PartitionMoveInterruption.test_cancelling_partition_move.replication_factor=1.unclean_abort=True.recovery=no_recovery.compacted=False"
"rptest.tests.cluster_health_overview_test.ClusterHealthOverviewTest.cluster_health_overview_baseline_test"
"rptest.tests.compatibility.kafka_streams_test.KafkaStreamsWikipedia.test_kafka_streams_wikipedia"
"rptest.tests.compatibility.sarama_test.SaramaTest.test_sarama_consumergroup"
"rptest.tests.partition_move_interruption_test.PartitionMoveInterruption.test_cancelling_partition_move_node_down"
"rptest.tests.controller_availability_test.ControllerAvailabilityTest.test_controller_availability_with_nodes_down.cluster_size=4.stop=minority"
"rptest.tests.partition_move_interruption_test.PartitionMoveInterruption.test_cancelling_partition_move_x_core.replication_factor=1.unclean_abort=False.recovery=restart_recovery.compacted=True"
"rptest.tests.custom_topic_assignment_test.CustomTopicAssignmentTest.test_custom_assignment_validation"
"rptest.tests.partition_move_interruption_test.PartitionMoveInterruption.test_cancelling_partition_move_x_core.replication_factor=1.unclean_abort=True.recovery=restart_recovery.compacted=True"
"rptest.tests.partition_move_interruption_test.PartitionMoveInterruption.test_cancelling_partition_move_x_core.replication_factor=3.unclean_abort=False.recovery=restart_recovery.compacted=True"
"rptest.tests.partition_move_interruption_test.PartitionMoveInterruption.test_cancelling_partition_move.replication_factor=3.unclean_abort=True.recovery=no_recovery.compacted=False"
"rptest.tests.delete_records_test.DeleteRecordsTest.test_delete_records_concurrent_truncations.cloud_storage_enabled=False.truncate_point=random_offset"
"rptest.tests.partition_move_interruption_test.PartitionMoveInterruption.test_cancelling_partition_move_x_core.replication_factor=3.unclean_abort=True.recovery=restart_recovery.compacted=True"
"rptest.tests.partition_movement_test.PartitionMovementTest.test_movement_tracking_api"
"rptest.tests.delete_records_test.DeleteRecordsTest.test_delete_records_concurrent_truncations.cloud_storage_enabled=True.truncate_point=random_offset"
"rptest.tests.scaling_up_test.ScalingUpTest.test_adding_multiple_nodes_to_the_cluster.partition_count=1"
"rptest.tests.scaling_up_test.ScalingUpTest.test_adding_nodes_to_cluster.partition_count=20"
"rptest.tests.audit_log_test.AuditLogTestOauth.test_admin_oauth"
"rptest.tests.compatibility.kafka_streams_test.KafkaStreamsAnomalyDetection.test_kafka_streams"
"rptest.tests.consumer_group_test.ConsumerGroupTest.test_basic_group_join.static_members=True"
"rptest.tests.consumer_group_test.ConsumerGroupTest.test_consumer_rejoin.static_members=True"
"rptest.tests.e2e_iam_role_test.AWSRoleFetchTests.test_write"
"rptest.tests.follower_fetching_test.FollowerFetchingTest.test_basic_follower_fetching.read_from_object_store=False"
"rptest.tests.nodes_decommissioning_test.NodesDecommissioningTest.test_decommissioning_finishes_after_manual_cancellation.delete_topic=True"
"rptest.tests.nodes_decommissioning_test.NodesDecommissioningTest.test_decommissioning_working_node.delete_topic=False.tick_interval=5000"
"rptest.tests.nodes_decommissioning_test.NodesDecommissioningTest.test_flipping_decommission_recommission.node_is_alive=True"
"rptest.tests.nodes_decommissioning_test.NodesDecommissioningTest.test_recommissioning_node"
"rptest.tests.rack_aware_replica_placement_test.RackAwarePlacementTest.test_node_config_update"
"rptest.tests.redpanda_multi_region_test.RedpandaNetworkDelayTest.test_basic_traffic_shaping"
"rptest.tests.nodes_decommissioning_test.NodesDecommissioningTest.test_recycle_all_nodes.auto_assign_node_id=True"
"rptest.tests.topic_creation_test.RecreateTopicMetadataTest.test_recreated_topic_metadata_are_valid.replication_factor=5"
"rptest.tests.acls_test.AccessControlListTestUpgrade.test_describe_acls.use_tls=False.use_sasl=False.enable_authz=False.authn_method=sasl.client_auth=False"
"rptest.tests.full_disk_test.FullDiskTest.test_full_disk_no_produce"
"rptest.tests.acls_test.AccessControlListTestUpgrade.test_describe_acls.use_tls=False.use_sasl=False.enable_authz=None.authn_method=sasl.client_auth=False"
"rptest.tests.acls_test.AccessControlListTestUpgrade.test_describe_acls.use_tls=False.use_sasl=False.enable_authz=True.authn_method=sasl.client_auth=False"
"rptest.tests.partition_force_reconfiguration_test.PartitionForceReconfigurationTest.test_reconfiguring_with_dead_node.controller_snapshots=True"
"rptest.tests.leaders_info_api_test.LeadersInfoApiEndToEndTest.reset_leaders_info_end_to_end_test"
"rptest.tests.acls_test.AccessControlListTestUpgrade.test_describe_acls.use_tls=False.use_sasl=True.enable_authz=False.authn_method=sasl.client_auth=False"
"rptest.tests.partition_movement_test.PartitionMovementTest.test_dynamic.num_to_upgrade=0"
"rptest.tests.partition_force_reconfiguration_test.PartitionForceReconfigurationTest.test_node_wise_recovery.dead_node_count=2"
"rptest.tests.partition_state_api_test.PartitionStateAPItest.test_controller_partition_state"
"rptest.tests.rpk_group_test.RpkGroupCommandsTest.test_group_describe"
"rptest.tests.acls_test.AccessControlListTestUpgrade.test_describe_acls.use_tls=False.use_sasl=True.enable_authz=None.authn_method=sasl.client_auth=False"
"rptest.tests.topic_creation_test.CreateTopicReplicaDistributionTest.test_topic_aware_distribution"
"rptest.tests.topic_creation_test.TopicRecreateTest.test_topic_recreation_while_producing.workload=ACKS_ALL.cleanup_policy=compact"
"rptest.tests.acls_test.AccessControlListTestUpgrade.test_describe_acls.use_tls=False.use_sasl=True.enable_authz=True.authn_method=sasl.client_auth=False"
"rptest.tests.audit_log_test.AuditLogTestAdminApi.test_audit_log_metrics"
"rptest.tests.acls_test.AccessControlListTestUpgrade.test_describe_acls.use_tls=True.use_sasl=False.enable_authz=False.authn_method=sasl.client_auth=False"
"rptest.tests.acls_test.AccessControlListTestUpgrade.test_describe_acls.use_tls=True.use_sasl=False.enable_authz=None.authn_method=sasl.client_auth=False"
"rptest.tests.timely_shutdown_test.ShutdownTest.test_timely_shutdown_with_failures"
"rptest.tests.cloud_storage_chunk_read_path_test.CloudStorageChunkReadTest.test_prefetch_chunks.prefetch=3"
"rptest.tests.acls_test.AccessControlListTestUpgrade.test_describe_acls.use_tls=True.use_sasl=False.enable_authz=True.authn_method=sasl.client_auth=False"
"rptest.tests.cloud_storage_chunk_read_path_test.CloudStorageChunkReadTest.test_read_when_chunk_api_disabled"
"rptest.tests.cluster_config_test.ClusterConfigNodeAddTest.test_node_add_with_snapshot"
"rptest.tests.acls_test.AccessControlListTestUpgrade.test_describe_acls.use_tls=True.use_sasl=True.enable_authz=False.authn_method=sasl.client_auth=False"
"rptest.tests.write_caching_fi_e2e_test.WriteCachingFailureInjectionE2ETest.test_crash_all_with_consumer_group"
"rptest.tests.controller_snapshot_test.ControllerSnapshotTest.test_bootstrap.auto_assign_node_ids=True"
"rptest.tests.cluster_recovery_test.ClusterRecoveryTest.test_bootstrap_with_recovery"
"rptest.tests.data_transforms_test.DataTransformsLeadershipChangingTest.test_leadership_changing_randomly"
"rptest.tests.cpu_profiler_admin_api_test.CPUProfilerAdminAPITest.test_get_cpu_profile_with_override"
"rptest.tests.data_transforms_test.DataTransformsTest.test_compression.compression_type=CompressionTypes.NONE"
"rptest.tests.data_transforms_test.DataTransformsTest.test_consume_from_offset.offset=.9223372036854775807"
"rptest.tests.data_transforms_test.DataTransformsTest.test_consume_from_end"
"rptest.tests.data_transforms_test.DataTransformsTest.test_consume_off_end.offset=.33276193569000"
"rptest.tests.data_transforms_test.DataTransformsTest.test_identity.transactional=True.wait_running=False"
"rptest.tests.e2e_shadow_indexing_test.EndToEndHydrationTimeoutTest.test_hydration_completes_when_consumer_killed"
"rptest.tests.fetch_long_poll_test.FetchTest.fetch_long_poll_test"
"rptest.tests.partition_reassignments_test.PartitionReassignmentsTest.test_disable_alter_reassignment_api"
"rptest.tests.e2e_shadow_indexing_test.EndToEndSpilloverTest.test_spillover.cloud_storage_type=CloudStorageType.S3"
"rptest.tests.e2e_shadow_indexing_test.EndToEndShadowIndexingTest.test_reset_from_cloud.cloud_storage_type=CloudStorageType.S3"
"rptest.tests.recovery_mode_test.DisablingPartitionsTest.test_apis"
"rptest.tests.rpk_topic_test.RpkToolTest.test_consume_from_partition"
"rptest.tests.redpanda_oauth_test.RedpandaOIDCTest.test_init"
"rptest.tests.redpanda_oauth_test.OIDCReauthTest.test_oidc_reauth"
"rptest.tests.schema_registry_test.SchemaRegistryAutoAuthTest.test_concurrent_writes"
"rptest.tests.schema_registry_test.SchemaRegistryAutoAuthTest.test_schema_id_validation.protocol=AVRO.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.RECORD_NAME.payload_class=com.redpanda.CompressiblePayload.compression_type=CompressionTypes.NONE"
"rptest.tests.redpanda_oauth_test.RedpandaOIDCTlsTest.test_init"
"rptest.tests.rpk_topic_test.RpkToolTest.test_produce"
"rptest.tests.schema_registry_test.SchemaRegistryAutoAuthTest.test_schema_id_validation.protocol=AVRO.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.TOPIC_NAME.payload_class=com.redpanda.A.B.C.D.NestedPayload.compression_type=CompressionTypes.NONE"
"rptest.tests.schema_registry_test.SchemaRegistryAutoAuthTest.test_schema_id_validation.protocol=AVRO.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.TOPIC_NAME.payload_class=com.redpanda.Payload.compression_type=CompressionTypes.NONE"
"rptest.tests.schema_registry_test.SchemaRegistryAutoAuthTest.test_schema_id_validation.protocol=AVRO.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.TOPIC_RECORD_NAME.payload_class=com.redpanda.CompressiblePayload.compression_type=CompressionTypes.NONE"
"rptest.tests.schema_registry_test.SchemaRegistryAutoAuthTest.test_schema_id_validation.protocol=JSON.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.RECORD_NAME.payload_class=com.redpanda.A.B.C.D.NestedPayload.compression_type=CompressionTypes.NONE"
"rptest.tests.schema_registry_test.SchemaRegistryAutoAuthTest.test_schema_id_validation.protocol=JSON.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.RECORD_NAME.payload_class=com.redpanda.Payload.compression_type=CompressionTypes.NONE"
"rptest.tests.schema_registry_test.SchemaRegistryAutoAuthTest.test_schema_id_validation.protocol=JSON.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.TOPIC_NAME.payload_class=com.redpanda.CompressiblePayload.compression_type=CompressionTypes.NONE"
"rptest.tests.schema_registry_test.SchemaRegistryAutoAuthTest.test_schema_id_validation.protocol=JSON.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.TOPIC_RECORD_NAME.payload_class=com.redpanda.A.B.C.D.NestedPayload.compression_type=CompressionTypes.NONE"
"rptest.tests.schema_registry_test.SchemaRegistryAutoAuthTest.test_schema_id_validation.protocol=JSON.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.TOPIC_RECORD_NAME.payload_class=com.redpanda.Payload.compression_type=CompressionTypes.NONE"
"rptest.tests.schema_registry_test.SchemaRegistryAutoAuthTest.test_schema_id_validation.protocol=PROTOBUF.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.RECORD_NAME.payload_class=com.redpanda.CompressiblePayload.compression_type=CompressionTypes.NONE"
"rptest.tests.schema_registry_test.SchemaRegistryAutoAuthTest.test_schema_id_validation.protocol=PROTOBUF.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.TOPIC_NAME.payload_class=com.redpanda.A.B.C.D.NestedPayload.compression_type=CompressionTypes.NONE"
"rptest.tests.schema_registry_test.SchemaRegistryAutoAuthTest.test_schema_id_validation.protocol=PROTOBUF.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.TOPIC_RECORD_NAME.payload_class=com.redpanda.CompressiblePayload.compression_type=CompressionTypes.NONE"
"rptest.tests.schema_registry_test.SchemaRegistryAutoAuthTest.test_serde_client.protocol=PROTOBUF.client_type=Golang"
"rptest.tests.schema_registry_test.SchemaRegistryAutoAuthTest.test_schema_id_validation.protocol=PROTOBUF.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.TOPIC_NAME.payload_class=com.redpanda.Payload.compression_type=CompressionTypes.NONE"
"rptest.tests.schema_registry_test.SchemaRegistryAutoAuthTest.test_serde_client.protocol=PROTOBUF.client_type=Python.skip_known_types=True"
"rptest.tests.e2e_topic_recovery_test.EndToEndTopicRecovery.test_restore_with_config_batches.num_messages=2.cloud_storage_type=CloudStorageType.ABS"
"rptest.tests.schema_registry_test.SchemaRegistryTest.test_concurrent_writes"
"rptest.tests.schema_registry_test.SchemaRegistryTest.test_schema_id_validation.protocol=AVRO.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.RECORD_NAME.payload_class=com.redpanda.CompressiblePayload.compression_type=CompressionTypes.NONE"
"rptest.tests.schema_registry_test.SchemaRegistryTest.test_schema_id_validation.protocol=AVRO.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.TOPIC_NAME.payload_class=com.redpanda.Payload.compression_type=CompressionTypes.NONE"
"rptest.tests.schema_registry_test.SchemaRegistryTest.test_schema_id_validation.protocol=AVRO.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.TOPIC_RECORD_NAME.payload_class=com.redpanda.CompressiblePayload.compression_type=CompressionTypes.NONE"
"rptest.tests.schema_registry_test.SchemaRegistryTest.test_schema_id_validation.protocol=JSON.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.RECORD_NAME.payload_class=com.redpanda.A.B.C.D.NestedPayload.compression_type=CompressionTypes.NONE"
"rptest.tests.schema_registry_test.SchemaRegistryTest.test_schema_id_validation.protocol=JSON.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.RECORD_NAME.payload_class=com.redpanda.Payload.compression_type=CompressionTypes.NONE"
"rptest.tests.schema_registry_test.SchemaRegistryTest.test_schema_id_validation.protocol=JSON.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.TOPIC_NAME.payload_class=com.redpanda.CompressiblePayload.compression_type=CompressionTypes.NONE"
"rptest.tests.schema_registry_test.SchemaRegistryTest.test_schema_id_validation.protocol=JSON.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.TOPIC_RECORD_NAME.payload_class=com.redpanda.A.B.C.D.NestedPayload.compression_type=CompressionTypes.NONE"
"rptest.tests.schema_registry_test.SchemaRegistryTest.test_schema_id_validation.protocol=JSON.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.TOPIC_RECORD_NAME.payload_class=com.redpanda.Payload.compression_type=CompressionTypes.NONE"
"rptest.tests.schema_registry_test.SchemaRegistryTest.test_schema_id_validation.protocol=PROTOBUF.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.RECORD_NAME.payload_class=com.redpanda.CompressiblePayload.compression_type=CompressionTypes.NONE"
"rptest.tests.schema_registry_test.SchemaRegistryTest.test_schema_id_validation.protocol=PROTOBUF.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.TOPIC_NAME.payload_class=com.redpanda.A.B.C.D.NestedPayload.compression_type=CompressionTypes.NONE"
"rptest.tests.schema_registry_test.SchemaRegistryTest.test_schema_id_validation.protocol=PROTOBUF.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.TOPIC_NAME.payload_class=com.redpanda.Payload.compression_type=CompressionTypes.NONE"
"rptest.tests.schema_registry_test.SchemaRegistryTest.test_schema_id_validation.protocol=PROTOBUF.client_type=Python.validate_schema_id=True.subject_name_strategy=SubjectNameStrategyCompat.TOPIC_RECORD_NAME.payload_class=com.redpanda.CompressiblePayload.compression_type=CompressionTypes.NONE"
"rptest.tests.schema_registry_test.SchemaRegistryTest.test_serde_client.protocol=PROTOBUF.client_type=Golang"
"rptest.tests.schema_registry_test.SchemaRegistryTest.test_serde_client.protocol=PROTOBUF.client_type=Python.skip_known_types=True"
"rptest.tests.shadow_indexing_compacted_topic_test.ShadowIndexingCompactedTopicTest.test_upload.cloud_storage_type=CloudStorageType.ABS"
"rptest.tests.tiered_storage_model_test.TieredStorageTest.test_tiered_storage.cloud_storage_type_and_url_style=.CloudStorageType.S3.1.path.test_case=.TS_Read==True.SpilloverManifestUploaded==True"
"rptest.tests.tiered_storage_model_test.TieredStorageTest.test_tiered_storage.cloud_storage_type_and_url_style=.CloudStorageType.S3.1.path.test_case=.TS_Delete==True.SpilloverManifestUploaded==True.TS_Spillover_ManifestDeleted==True"
"rptest.tests.tiered_storage_model_test.TieredStorageTest.test_tiered_storage.cloud_storage_type_and_url_style=.CloudStorageType.ABS.2.virtual_host.test_case=.TS_Read==True.TS_Timequery==True"
"rptest.tests.timequery_test.TimeQueryTest.test_timequery_below_start_offset.spillover=False"
"rptest.tests.tiered_storage_model_test.TieredStorageTest.test_tiered_storage.cloud_storage_type_and_url_style=.CloudStorageType.S3.1.path.test_case=.TS_Read==True.TS_TxRangeMaterialized==True"
"rptest.tests.timequery_test.TimeQueryTest.test_timequery.cloud_storage=False.batch_cache=False.spillover=False"
"rptest.tests.topic_delete_test.TopicDeleteAfterMovementTest.topic_delete_orphan_files_after_move_test"
"rptest.tests.tiered_storage_model_test.TieredStorageTest.test_tiered_storage.cloud_storage_type_and_url_style=.CloudStorageType.S3.1.virtual_host.test_case=.TS_Read==True.AdjacentSegmentMergerReupload==True.SpilloverManifestUploaded==True"
"rptest.tests.timequery_test.TimeQueryTest.test_timequery_with_spillover_gc_delayed"
"rptest.tests.tiered_storage_model_test.TieredStorageTest.test_tiered_storage.cloud_storage_type_and_url_style=.CloudStorageType.S3.1.virtual_host.test_case=.TS_Read==True.TS_Timequery==True"
"rptest.tests.topic_recovery_test.TopicRecoveryTest.test_missing_partition.cloud_storage_type=CloudStorageType.S3"
"rptest.tests.topic_recovery_test.TopicRecoveryTest.test_admin_api_recovery.cloud_storage_type=CloudStorageType.S3"
"rptest.tests.topic_recovery_test.TopicRecoveryTest.test_fast1.cloud_storage_type=CloudStorageType.S3"
"rptest.tests.usage_test.UsageTest.test_usage_collection_restart"
"rptest.tests.topic_recovery_test.TopicRecoveryTest.test_prevent_recovery.cloud_storage_type=CloudStorageType.S3"
"rptest.transactions.compacted_verifier_test.CompactedVerifierTest.test_idempotency"
"rptest.tests.acls_test.AccessControlListTestUpgrade.test_describe_acls.use_tls=True.use_sasl=True.enable_authz=None.authn_method=sasl.client_auth=False"
"rptest.tests.acls_test.AccessControlListTestUpgrade.test_mtls_principal.rules=RULE.O=Redpanda.CN=.1.L.fail=False"
"rptest.tests.acls_test.AccessControlListTestUpgrade.test_describe_acls.use_tls=True.use_sasl=True.enable_authz=True.authn_method=sasl.client_auth=False"
"rptest.tests.write_caching_fi_test.WriteCachingFailureInjectionTest.test_unavoidable_data_loss"
"rptest.tests.acls_test.AccessControlListTestUpgrade.test_mtls_principal.rules=RULE.O=Redpanda.CN=.redpanda.service.admin.admin.1.RULE.O=.CN=.1.fail=True"
"rptest.tests.admin_api_auth_test.AdminApiAuthEnablementTest.test_combined_request"
"rptest.tests.admin_api_auth_test.AdminApiAuthTest.test_regular_user_access"
"rptest.tests.topic_recovery_test.TopicRecoveryTest.test_time_based_retention.cloud_storage_type=CloudStorageType.S3"
"rptest.tests.alter_topic_configuration_test.AlterTopicConfiguration.test_altering_topic_configuration.property=message.timestamp.type.value=LogAppendTime"
"rptest.tests.admin_uuid_operations_test.AdminUUIDOperationsTest.test_overriding_node_id"
"rptest.tests.alter_topic_configuration_test.AlterTopicConfiguration.test_configuration_properties_kafka_config_allowlist"
"rptest.tests.adjacent_segment_merging_test.AdjacentSegmentMergingTest.test_reupload_of_local_segments.acks=-1.cloud_storage_type=CloudStorageType.ABS"
"rptest.tests.alter_topic_configuration_test.AlterTopicConfiguration.test_shadow_indexing_config"
"rptest.tests.archival_test.ArchivalTest.test_compaction_topic_config_change.cloud_storage_type=CloudStorageType.ABS"
"rptest.tests.admin_uuid_operations_test.AdminUUIDOperationsTest.test_force_uuid_override.mode=TestMode.NO_OVERRIDE"
"rptest.tests.archival_test.ArchivalTest.test_one_node_reconnect.cloud_storage_type=CloudStorageType.ABS"
"rptest.tests.archival_test.ArchivalTest.test_connection_flicker.cloud_storage_type=CloudStorageType.ABS"
"rptest.tests.archival_test.ArchivalTest.test_retention_archival_coordination.acks=-1.cloud_storage_type=CloudStorageType.ABS"
"rptest.tests.archival_test.ArchivalTest.test_all_partitions_leadership_transfer.cloud_storage_type=CloudStorageType.ABS"
"rptest.tests.client_quota_deprecated_configs_test.ClientQuotaDeprecatedConfigs_ConfigUpdateTest.test_config_update"
"rptest.tests.archival_test.ArchivalTest.test_write.cloud_storage_type_and_url_style=.CloudStorageType.ABS.2.virtual_host"
"rptest.tests.cluster_bootstrap_test.ClusterBootstrapNew.test_three_node_bootstrap.num_seeds=1.auto_assign_node_ids=False.empty_seed_starts_cluster=False"
"rptest.tests.cluster_bootstrap_test.ClusterBootstrapNew.test_three_node_bootstrap.num_seeds=2.auto_assign_node_ids=False.empty_seed_starts_cluster=False"
"rptest.tests.cluster_bootstrap_test.ClusterBootstrapNew.test_three_node_bootstrap.num_seeds=3.auto_assign_node_ids=False.empty_seed_starts_cluster=False"
"rptest.tests.admin_operations_test.AdminOperationsTest.test_admin_operations"
"rptest.tests.archival_test.ArchivalTest.test_single_partition_leadership_transfer.cloud_storage_type=CloudStorageType.ABS"
"rptest.tests.archival_test.ArchivalTest.test_timeboxed_uploads.acks=0.cloud_storage_type=CloudStorageType.ABS"
"rptest.tests.cluster_config_test.ClusterConfigAliasTest.test_aliasing.prop_set=PropertyAliasData.primary_name=.cloud_storage_graceful_transfer_timeout_ms.aliased_name=.cloud_storage_graceful_transfer_timeout.redpanda_version=.23.2.test_values=.1234.1235.1236.expect_restart=False"
"rptest.tests.cluster_config_test.ClusterConfigTest.test_alter_configs"
"rptest.tests.cluster_config_test.ClusterConfigTest.test_incremental_alter_configs"
"rptest.tests.cluster_config_test.ClusterConfigTest.test_invalid_settings.key=log_message_timestamp_type.value=31415"
"rptest.tests.cluster_config_test.ClusterConfigTest.test_invalid_settings.key=metadata_dissemination_retries.value=rhubarb"
"rptest.tests.cluster_config_test.ClusterConfigTest.test_rpk_import_sparse.all=False"
"rptest.tests.cluster_config_test.ClusterConfigTest.test_rpk_status"
"rptest.tests.cluster_config_test.DevelopmentFeatureTest.test_cannot_disable"
"rptest.tests.cluster_config_test.DevelopmentFeatureTest.test_reject_invalid_enable_key"
"rptest.tests.cluster_metadata_test.MetadataTest.test_metadata_request_does_not_contain_failed_node.failure=stop.node=controller"
"rptest.tests.cluster_metrics_test.ClusterMetricsTest.cluster_metrics_reported_only_by_leader_test"
"rptest.tests.cluster_quota_test.ClusterRateQuotaTest.test_client_group_consume_rate_throttle_mechanism"
"rptest.tests.cluster_quota_test.ClusterRateQuotaTest.test_client_response_throttle_mechanism"
"rptest.tests.configuration_update_test.ConfigurationUpdateTest.test_update_advertised_kafka_api_on_all_nodes"
"rptest.tests.compacted_term_rolled_recovery_test.CompactionTermRollRecoveryTest.test_compact_term_rolled_recovery"
"rptest.tests.connection_virtualizing_test.TestVirtualConnections.test_no_head_of_line_blocking.different_clusters=False.different_connections=False"
"rptest.tests.data_migrations_api_test.DataMigrationsApiTest.test_creating_with_topic_no_remote_writes"
"rptest.tests.controller_erase_test.ControllerEraseTest.test_erase_controller_log.partial=False"
"rptest.tests.data_transforms_test.DataTransformsTest.test_consume_junk_off.offset=.1727917147729.131"
"rptest.tests.crl_test.CertificateRevocationTest.test_pp_api"
"rptest.tests.data_transforms_test.DataTransformsLoggingTest.test_tunable_configs"
"rptest.tests.data_transforms_test.DataTransformsTest.test_tracked_offsets_cleaned_up"
"rptest.tests.delete_records_test.DeleteRecordsTest.test_delete_records_compacted_topic.cloud_storage_enabled=False"
"rptest.tests.delete_records_test.DeleteRecordsTest.test_delete_records_segment_deletion.cloud_storage_enabled=False.truncate_point=at_high_watermark"
"rptest.tests.delete_records_test.DeleteRecordsTest.test_delete_records_segment_deletion.cloud_storage_enabled=False.truncate_point=start_offset"
"rptest.tests.delete_records_test.DeleteRecordsTest.test_delete_records_segment_deletion.cloud_storage_enabled=True.truncate_point=random_offset"
"rptest.tests.enterprise_features_license_test.EnterpriseFeaturesTest.test_license_violation.feature=Features.core_balancing_continuous.with_license=False"
"rptest.tests.enterprise_features_license_test.EnterpriseFeaturesTest.test_license_violation.feature=Features.oidc.with_license=False"
"rptest.tests.describe_producers_test.DescribeProducersTest.test_describe_idempotent_producers"
"rptest.tests.enterprise_features_license_test.EnterpriseFeaturesTest.test_license_violation.feature=Features.rbac.with_license=False"
"rptest.tests.data_transforms_test.DataTransformsLoggingMetricsTest.test_manager_metrics_values"
"rptest.tests.full_disk_test.DiskStatsOverrideTest.test_disk_stat_free_delta_overrides"
"rptest.tests.delete_records_test.DeleteRecordsTest.test_delete_records_topic_start_delta.cloud_storage_enabled=False"
"rptest.tests.fetch_after_deletion_test.FetchAfterDeleteTest.test_fetch_after_committed_offset_was_removed.transactions_enabled=False"
"rptest.tests.internal_topic_protection_test.InternalTopicProtectionTest.kafka_nodelete_topics_test.client_type=rpk"
"rptest.tests.group_membership_test.ListGroupsReplicationFactorTest.test_list_groups_has_no_duplicates"
"rptest.tests.librdkafka_test.LibrdkafkaTest.test_create_bad_acl"
"rptest.tests.limits_test.LimitsTest.test_kafka_request_max_size"
"rptest.tests.internal_topic_protection_test.InternalTopicProtectionTest.kafka_protections_disable_config_test.protect_config=kafka_noproduce_topics"
"rptest.tests.full_disk_test.WriteRejectTest.test_write_rejection"
"rptest.tests.controller_partition_invariants_test.ControllerLogInvariantsTest.test_controller_is_not_compacted_nor_deleted.cluster_cleanup_policy=delete"
"rptest.tests.maintenance_test.MaintenanceTest.test_exclusive_maintenance.use_rpk=False"
"rptest.tests.maintenance_test.MaintenanceTest.test_maintenance_mode_of_stopped_node.use_rpk=False"
"rptest.tests.maintenance_test.MaintenanceTest.test_maintenance_with_single_replicas.use_rpk=False"
"rptest.tests.log_segment_ms_test.SegmentMsTest.test_segment_rolling_with_retention"
"rptest.tests.leadership_transfer_test.AutomaticLeadershipBalancingTest.test_automatic_rebalance"
"rptest.tests.metrics_test.MetricsTest.test_aggregate_metrics.aggregate_metrics=False"
"rptest.tests.node_id_assignment_test.NodeIdAssignment.test_basic_assignment"
"rptest.tests.node_status_test.NodeStatusStartupTest.test_late_joiner"
"rptest.tests.pandaproxy_test.PandaProxyAutoAuthTest.test_consumer_group_binary_v2"
"rptest.tests.pandaproxy_test.PandaProxyAutoAuthTest.test_produce_topic"
"rptest.tests.pandaproxy_test.PandaProxyAutoAuthTest.test_fetch_topic_validation"
"rptest.tests.pandaproxy_test.PandaProxyBasicAuthTest.test_fetch_topic"
"rptest.tests.pandaproxy_test.PandaProxyAutoAuthTest.test_restarts.move_controller_leader=False"
"rptest.tests.pandaproxy_test.PandaProxyCompressedBatchesTest.test_fetch"
"rptest.tests.pandaproxy_test.PandaProxyBasicAuthTest.test_produce_topic"
"rptest.tests.pandaproxy_test.PandaProxyInvalidInputsTest.test_invalid_fetch_consumer_assignments"
"rptest.tests.pandaproxy_test.PandaProxyInvalidInputsTest.test_invalid_subscribe_consumer"
"rptest.tests.pandaproxy_test.PandaProxyTest.test_consumer_group_binary_v2"
"rptest.tests.pandaproxy_test.PandaProxyMTLSAndBasicAuthTest.test_mtls_and_basic_auth"
"rptest.tests.pandaproxy_test.PandaProxyTest.test_fetch_topic_validation"
"rptest.tests.pandaproxy_test.PandaProxyTest.test_produce_topic"
"rptest.tests.pandaproxy_test.PandaProxyTest.test_subscribe_consumer_validation"
"rptest.tests.offset_retention_test.OffsetRetentionTest.test_offset_expiration"
"rptest.tests.partition_movement_test.PartitionMovementTest.test_invalid_destination.num_to_upgrade=0"
"rptest.tests.raft_availability_test.RaftAvailabilityTest.test_controller_node_isolation"
"rptest.tests.raft_availability_test.RaftAvailabilityTest.test_one_node_down"
"rptest.tests.prefix_truncate_recovery_test.PrefixTruncateRecoveryTest.test_prefix_truncate_recovery.acks=1.start_empty=True"
"rptest.tests.rbac_test.RBACTest.test_invalid_create_role"
"rptest.tests.rbac_test.RBACEndToEndTest.test_delete_role_acls.delete_acls=None"
"rptest.tests.rbac_test.RBACTest.test_members_endpoint"
"rptest.tests.redpanda_kerberos_test.RedpandaKerberosLicenseTest.test_license_nag"
"rptest.tests.redpanda_kerberos_test.RedpandaKerberosRulesTesting.test_kerberos_mapping_rules.rules=.RULE.2.1testbad.0.client.RULE.1.1testgood.client.s.client.1redpanda.U.DEFAULT.kerberos_principal=client.rp_user=TESTGOODREDPANDA.expected_topics=.restricted.always_visible.acl=.restricted.TESTGOODREDPANDA.always_visible"
"rptest.tests.log_segment_ms_test.SegmentMsTest.test_segment_not_rolling.use_alter_cfg=False.server_cfg=60000.topic_cfg=-1"
"rptest.tests.rbac_test.RolePersistenceTest.test_role_survives_restart"
"rptest.tests.redpanda_startup_test.RedpandaStartupTest.test_startup"
"rptest.tests.log_segment_ms_test.SegmentMsTest.test_segment_rolling.use_alter_cfg=True.server_cfg=60000.topic_cfg=None"
"rptest.tests.resource_limits_test.ResourceLimitsTest.test_fd_limited"
"rptest.tests.retention_policy_test.RetentionPolicyTest.test_changing_topic_retention.property=retention.bytes.acks=-1.cloud_storage_type=CloudStorageType.ABS"
"rptest.tests.retention_policy_test.ShadowIndexingCloudRetentionTest.test_cloud_retention_deleted_segments_count.cloud_storage_type=CloudStorageType.S3"
"rptest.tests.rpk_cluster_test.RpkClusterTest.test_upload_cluster_license_error"
"rptest.tests.retention_policy_test.RetentionPolicyTest.test_changing_topic_retention.property=retention.ms.acks=-1.cloud_storage_type=CloudStorageType.ABS"
"rptest.tests.rpk_config_test.RpkConfigTest.test_config_set_json"
"rptest.tests.rpk_profile_test.RpkProfileTest.test_create_profile_from_redpanda"
"rptest.tests.retention_policy_test.RetentionPolicyTest.test_changing_topic_retention_with_restart.cloud_storage_type=CloudStorageType.ABS"
"rptest.tests.rpk_cluster_test.RpkClusterTest.test_debug_bundle"
"rptest.tests.rpk_start_test.RpkRedpandaStartTest.test_bootstrap_then_start"
"rptest.tests.rpk_start_test.RpkRedpandaStartTest.test_rpc_tls_list"
"rptest.tests.rpk_registry_test.RpkRegistryTest.test_produce_consume_json"
"rptest.tests.schema_registry_test.SchemaRegistryAutoAuthTest.test_compatibility_messages.schemas=.avro.avro_incompat.AVRO"
"rptest.tests.rpk_start_test.RpkRedpandaStartTest.test_simple_start_three_with_root"
"rptest.tests.schema_registry_test.SchemaRegistryAutoAuthTest.test_get_schema_id_subjects"
"rptest.tests.rpk_registry_test.RpkRegistryTest.test_registry_schema"
"rptest.tests.schema_registry_test.SchemaRegistryAutoAuthTest.test_post_compatibility_subject_version.dataset_type=JSON"
"rptest.tests.schema_registry_test.SchemaRegistryAutoAuthTest.test_post_subjects_subject_versions_metadata_ruleset"
"rptest.tests.schema_registry_test.SchemaRegistryAutoAuthTest.test_move_leader"
"rptest.tests.schema_registry_test.SchemaRegistryBasicAuthTest.test_delete_subject_last_clears_config"
"rptest.tests.schema_registry_test.SchemaRegistryAutoAuthTest.test_schemas_types"
"rptest.tests.schema_registry_test.SchemaRegistryBasicAuthTest.test_hard_delete_subject_deletes_schema"
"rptest.tests.schema_registry_test.SchemaRegistryConfluentClient.test_versions"
"rptest.tests.schema_registry_test.SchemaRegistryConfluentClient.test_register_get_lookup_schema.normalize_schemas=False"
"rptest.tests.schema_registry_test.SchemaRegistryTest.test_compatibility_messages.schemas=.avro.avro_incompat.AVRO"
"rptest.tests.schema_registry_test.SchemaRegistryTest.test_get_schema_id_subjects"
"rptest.tests.schema_registry_test.SchemaRegistryMTLSTest.test_mtls"
"rptest.tests.schema_registry_test.SchemaRegistryTest.test_post_compatibility_subject_version.dataset_type=JSON"
"rptest.tests.schema_registry_test.SchemaRegistryTest.test_move_leader"
"rptest.tests.schema_registry_test.SchemaRegistryTest.test_post_subjects_subject_versions_metadata_ruleset"
"rptest.tests.schema_registry_test.SchemaRegistryTest.test_schemas_types"
"rptest.tests.schema_registry_test.SchemaRegistryBasicAuthTest.test_post_subjects_subject_versions"
"rptest.tests.scram_test.InvalidNewUserStrings.test_invalid_alg"
"rptest.tests.scram_test.ScramTest.test_redirects.alternate_listener=False"
"rptest.tests.scram_test.ScramBootstrapUserTest.test_bootstrap_user.mechanism=SCRAM-SHA-256"
"rptest.tests.scramful_eos_test.ScramfulEosTest.test_idempotent_write_passes_1"
"rptest.tests.scramful_eos_test.ScramfulEosTest.test_tx_init_passes_2"
"rptest.tests.self_test_test.SelfTestTest.test_self_test_cancellable"
"rptest.tests.schema_registry_test.SchemaRegistryAutoAuthTest.test_restarts.move_controller_leader=False"
"rptest.tests.shadow_indexing_admin_api_test.SIAdminApiTest.test_topic_recovery_on_leader"
"rptest.tests.throughput_limits_snc_test.ThroughputLimitsSnc.test_configuration"
"rptest.tests.raft_availability_test.RaftAvailabilityTest.test_leader_restart"
"rptest.tests.tls_metrics_test.TLSMetricsTest.test_public_metrics"
"rptest.tests.tls_metrics_test.TLSMetricsTest.test_crc32c"
"rptest.tests.topic_creation_test.CreateTopicsResponseTest.test_create_topic_responses.partition_count=3.replication_factor=-1"
"rptest.tests.topic_creation_test.CreateTopicsTest.test_create_with_min_rf"
"rptest.tests.tls_version_test.TLSVersionTest.test_change_version.version=0"
"rptest.tests.schema_registry_test.SchemaRegistryTest.test_restarts.move_controller_leader=False"
"rptest.tests.shadow_indexing_firewall_test.ShadowIndexingFirewallTest.test_consume_from_blocked_s3.cloud_storage_type=CloudStorageType.S3"
"rptest.tests.topic_delete_test.TopicDeleteTest.topic_delete_orphan_files_test"
"rptest.tests.vcluster_topic_property_test.VirtualClusterTopicPropertyTest.test_basic_virtual_cluster_property"
"rptest.transactions.rpk_transaction_test.TxRpkTest.test_last_timestamp_of_describe_producers"
"rptest.transactions.transaction_kafka_api_test.TxKafkaApiTest.test_describe_transactions"
"rptest.tests.topic_recovery_test.TopicRecoveryTest.test_missing_topic_manifest.cloud_storage_type=CloudStorageType.S3"
"rptest.transactions.transactions_test.TransactionsTest.graceful_leadership_transfer_test"
"rptest.transactions.tx_admin_api_test.TxAdminTest.test_all_transactions"
"rptest.transactions.transactions_test.TransactionsTest.simple_test"
"rptest.transactions.tx_admin_api_test.TxAdminTest.test_delete_topic_from_ongoin_tx"
"rptest.transactions.tx_reads_writes_test.TxReadsWritesTest.test_reads_writes"
"rptest.transactions.transactions_test.TransactionsTest.check_sequence_table_cleaning_after_eviction_test"
"rptest.transactions.transactions_test.TransactionsStreamsTest.consumer_offsets_retention_test"
"rptest.tests.log_segment_ms_test.SegmentMsTest.test_segment_rolling.use_alter_cfg=False.server_cfg=300000.topic_cfg=None"
"rptest.tests.log_segment_ms_test.SegmentMsTest.test_segment_rolling.use_alter_cfg=False.server_cfg=None.topic_cfg=300000"
```
<!--090118ac-12a2-4891-97af-6de8ef158dbd-->

vbotbuildovich commented:
<!--RETRIED FAILURES-->
ducktape was retried in https://buildkite.com/redpanda/redpanda/builds/55693#01924fe1-764a-435b-b947-38510ef6ad66
<!--7c3ee1f0-0f05-4ee1-8177-cae190abe6ca-->

BenPope commented on src/v/config/configuration.cc:
```diff
@@ -1190,7 +1299,7 @@ configuration::configuration()
       "storage_max_concurrent_replay",
       "Maximum number of partitions' logs that will be replayed concurrently "
-      "at startup, or flushed concurrently on shutdown.",
+      "at startup, or flushed concurrently on shutdown. asdasdasdasdas",
```
:eyes:

dotnwat replied:
@Deflaimun

Deflaimun replied:
Oops, not sure how that got in there. Will remove! Thanks for the catch

Deflaimun replied:
Fixed!

BenPope reviewed (commented):
Having a poke with this over here: https://github.com/redpanda-data/redpanda/pull/23615

BenPope commented on src/v/config/configuration.cc:
```diff
@@ -325,39 +335,47 @@ configuration::configuration()
+      "Raft heartbeat RPC (remote procedure call) timeout. Raft uses a "
+      "heartbeat mechanism to maintain leadership authority and to trigger "
+      "leader elections. The `raft_heartbeat_interval_ms` is a periodic "
+      "heartbeat sent by the partition leader to all followers to assert its "
```
The easy fix for the ducktape tests is to avoid the word "assert"

Deflaimun replied:
should we reword to remove "assert" ?

dotnwat replied:
i think we should add exceptions for these cases in ducktape. we don't want to restrict the language we can use in descriptions.

BenPope replied:
> i think we should add exceptions for these cases in ducktape. we don't want to restrict the language we can use in descriptions.

I agree, which is what I [attempted here](https://github.com/redpanda-data/redpanda/pull/23619/commits/d43b9596dff8b9ccad0402effdf2bfcbc493e5d3#diff-3f567de33923dec6fa4ca0462aa968416b58c956c824dd007cff23498c342785R127) - but the NodeCrash regex still needs changing. I suggested an easy workaround.

I don't think the description of the config really needs to be rendered to the log at startup, the value should suffice.

BenPope reviewed (commented):
Please don't escape the `%` (`\%`), I count 4 of them.

Deflaimun commented:
Force-push:
 - Remove wrong text identified in https://github.com/redpanda-data/redpanda/pull/23347#discussion_r1785987540 
- Remove % escapes.

BenPope commented:
> Force-push:
> 
> * Remove wrong text identified in [config: update config descriptions to match docs #23347 (comment)](https://github.com/redpanda-data/redpanda/pull/23347#discussion_r1785987540)
> * Remove % escapes.

It doesn't lint anymore, the lines are too long.

Deflaimun commented:
force-push: remove the word "assert" from description string.

BenPope commented:
The [interesting diff](https://github.com/redpanda-data/redpanda/compare/84bc7c6984037a87c055477c52e2d3878d9b73c8..0b1ae30b82006046bb6dceca6ecefe0f934c47a4) between my review and now.

BenPope commented:
Known CI Failures:
* https://redpandadata.atlassian.net/browse/CORE-7777 (x2)
Unknown (possible new)
* In `test_describe_topics_with_documentation_and_types`: `doc_string == prop.doc_string`
